<?xml version="1.0" encoding="UTF-8"?>

<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p"
	xmlns:context="http://www.springframework.org/schema/context"
	xsi:schemaLocation="http://www.springframework.org/schema/beans 
	http://www.springframework.org/schema/beans/spring-beans-3.0.xsd 
	http://www.springframework.org/schema/context 
	http://www.springframework.org/schema/context/spring-context-3.0.xsd 
	http://www.springframework.org/schema/task 
	http://www.springframework.org/schema/task/spring-task-3.0.xsd">

	<context:component-scan base-package="cs" />
	<context:component-scan base-package="com.cardif.framework.excepcion" />
	<context:component-scan base-package="com.cardif.satelite.util" />

	<bean id="log4jInitialization" class="org.springframework.beans.factory.config.MethodInvokingFactoryBean">
		<property name="targetClass" value="org.springframework.util.Log4jConfigurer" />
		<property name="targetMethod" value="initLogging" />
		<property name="arguments">
			<list>
				<value>classpath:com\cardif\framework\recursos\log4j.properties</value>
			</list>
		</property>
	</bean>

	<bean id="applicationProperties" class="com.cardif.satelite.util.PropertiesUtil" init-method="loadProperties" />

	<bean id="errorProperties" class="com.cardif.framework.excepcion.PropertiesErrorUtil" init-method="loadProperties" />

</beans>